app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :comments
+  has_many :posts
+  has_many :likes
```

Comment:
Kindly specify the foreign keys for posts, comments, and likes.

#### app/models/user.rb
```
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```

Reply:
thank you @KDlamini. I have rectify it.